pysteps/blending/linear_blending.py

```diff
@@ -208,3 +258,70 @@ def forecast(
         R_blended = precip_nowcast
 
     return R_blended
+
+
+def get_ranked_salience(precip_nowcast, precip_nwp):
+    """
+    Calculate ranked salience
+
+    Args:
```

Comment:
please use the numpy docstring format here (as in the `forecast` method above)

https://numpydoc.readthedocs.io/en/latest/format.html